### PR TITLE
Add timeout parameter

### PR DESCRIPTION
**Pull request by Robotex**
These changes add the timeout parameter for #69 
I've just learnt the basics of Go from a quick read of their Playground tutorial, so I apologize for any mistake

### CODE REVIEW

**Comment by bradleyfalzon** [comment]
I'm not a maintainer, but the cli package does have a DurationFlag (https://godoc.org/github.com/urfave/cli#DurationFlag) which may be a better option so you don't need to do the cast to time.Duration.

File: main.go [hunk 1]
```diff
@@ -78,6 +79,12 @@ func main() {
 			Usage:  "use TLS and verify the remote",
 			EnvVar: "DOCKER_TLS_VERIFY",
 		},
+		cli.IntFlag{
+			Name:	"timeout",
+			Usage:	"timeout before container is forcefully stopped",
```
Comment (schnapster) [inline main.go]: I suggest saying "timeout **in seconds** before container is forcefully stopped", to be more clear

**Comment by stffabi** [comment]
@Robotex thanks for your contribution. I like the idea of @bradleyfalzon to use the duration flag and we should also add some documentation on the README.md.

Would it be possible for you to make these changes?

**Comment by Robotex** [comment]
I have made and committed the requested changes, however ATM I don't have the build environment set up so I cannot test them. Feel free to review and comment if you encounter any compilation errors.

File: README.md [hunk 2]
```diff
@@ -95,6 +95,7 @@ docker run --rm v2tec/watchtower --help
 * `--interval, -i` Poll interval (in seconds). This value controls how frequently watchtower will poll for new images. Defaults to 300 seconds (5 minutes).
 * `--schedule, -s` [Cron expression](https://godoc.org/github.com/robfig/cron#hdr-CRON_Expression_Format) in 6 fields (rather than the traditional 5) which defines when and how often to check for new images. Either `--interval` or the schedule expression could be defined, but not both. An example: `--schedule "0 0 4 * * *" ` 
 * `--no-pull` Do not pull new images. When this flag is specified, watchtower will not attempt to pull new images from the registry. Instead it will only monitor the local image cache for changes. Use this option if you are building new images directly on the Docker host without pushing them to a registry.
+* `--timeout` Timeout before the container is forcefully stopped. When set, this option will change the default (`10s`) wait time to the given value. An example: `--timeout 30s` will set the timeout to 30 seconds.
```
Comment (stffabi) [inline README.md]: What would you think about naming the option `--stop-timeout`? This makes it more clear what timeout this is.

File: main.go [hunk 3]
```diff
@@ -178,6 +185,7 @@ func before(c *cli.Context) error {
 
 	cleanup = c.GlobalBool("cleanup")
 	noRestart = c.GlobalBool("no-restart")
+	timeout = c.GlobalDuration("timeout")
```
Comment (stffabi) [inline main.go]: Timeout could be negative, we should check for that and print out an error.

**Review by stffabi — CHANGES_REQUESTED** [review]
Thanks @Robotex for your work and adjustments. I have only two little comments.

Would be great if you could take a look into those.

**Review by stffabi — APPROVED** [review]

**Comment by stffabi** [comment]
LGTM, thanks for your contribution. I'll just squash your commits and then merge it in.

**Comment by reloxx13** [comment]
pls check this: #195

--stop-timeout not found in armhf-0.3

**Pull request merged**